test/reporters/list.spec.js

```diff
@@ -138,10 +138,15 @@ describe('List reporter', function() {
       var actual = {a: 'actual'};
       var expected = {a: 'expected'};
       var checked = false;
+      var stringifyDiffs = 'stringifyDiffObjs';
       var err;
       test = {};
       runner.on = runner.once = function(event, callback) {
```

Comment:
If this is set to both `.on` and `.once`, aren't we calling it twice for same issue?

Reply:
I dont think so. `once` touches `start`+`end` events and `on` touches everything else.
As the assertion here doesn't need `start` or `end` I believe I can remove the `once`.